config.json.template

```diff
@@ -3,13 +3,19 @@
     "DATABASE_URL": "",
     "FROM_EMAIL": "",
     "EMAIL_SUBJECT": "Discord Verification",
-    "ROLE_NAME": "",
+    "VERIFIED_ROLE_NAME": "",
     "GUILD_ID": "",
     "WELCOME_CHANNEL_ID": "",
     "MEMBER_JOIN_MESSAGE": "",
     "EMAIL_REGEX": ".*@.*",
     "SMPT_JS_LOGIN_TOKEN": "",
     "CRYPTO_JSON_ALGORITHM": "aes256",
     "CRYPTO_JSON_ENCODING": "hex",
-    "CRYPTO_JSON_MEMBER_ENCRYPT_KEY": ""
+    "CRYPTO_JSON_MEMBER_ENCRYPT_KEY": "",
+    "SELF_JOIN_COMMAND": "!join",
+    "SELF_JOIN_REQUIRE_VERIFIED": true,
+    "SELF_JOIN_ROLE_NAMES": [
+    ],
+    "SELF_JOIN_ALIASES": {
```

Comment:
this too needs to be a string. 

Not sure how we can support aliases :( 

First idea is to just make it a json dump in a string that can be parsed on load?

Reply:
I think the easiest option is to strip it entirely :P KISS?

Reply:
Removed it for now, maybe something to do for a future feature